src/Microsoft.Azure.WebJobs.ServiceBus/Bindings/MessageSenderArgumentBindingProvider.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.ServiceBus.Core;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus.Bindings
+{
+    internal class MessageSenderArgumentBindingProvider : IQueueArgumentBindingProvider
```

Comment:
Could this be done with the v2 converter interfaces, rather than the v1 argumentbinder ?

Reply:
The entire binding needs to be ported to new model. That work is orthogonal to this fix.